vcr/signature/ldutils.go

```diff
@@ -28,12 +28,27 @@ import (
 	"net/url"
 )
 
+// JSONLDContextsConfig contains config for json-ld document loader
+type JSONLDContextsConfig struct {
```

Comment:
shouldn't this config be more nested, having JSONLD as a configuration level, to be more future proof (adding JSON-LD specific config)? So it becomes `vcr.jsonld.allowlist` and `vcr.jsonld.localmapping`?

Reply:
I guess it will be restructured anyways when moved to the engine.